packages/contracts/foundry.toml

```diff
@@ -5,7 +5,6 @@ out = 'out'
 libs = ['lib']
 script = 'scripts'
 solc_version = '0.8.19'
-evm_version = 'shanghai'
```

Comment:
why delete the version?  if i recall we need it this because we implemented SMTChecker

Reply:
I fixed it in the next commit. Was done in error